src/engineering.html

```diff
@@ -291,6 +291,22 @@ <h3><a href="https://www.linkedin.com/in/sam-simonds-485159b0/">Sam Simonds</a><
                 <p>Software engineer with a strong foundation in applications leveraging TypeScript / Next.js, JavaScipt, HTML and CSS. He's built web applications for start ups, healthcare organizations, and non-profits around the world.</p>
             </div>
         </div>
+        <div class="single-bio">
+          <a href="https://zfo.gg/">
+            <div class="thumb">
+              <div class="avatar-container">
+                <div class="avatar-head"></div>
+                <div class="avatar-body"></div>
+              </div>
+            </div>
+          </a>
+          <div class="bio-info">
+            <h3><a href="https://zfo.gg/">Zachary Fogg</a></h3>
+            <p>Specializing in frontend web, devops/infrastructure, and blockchain applications, he's built applications
```

Comment:
bro can you add a quick sentence at the end with your languages? Like straight up just `JS / TS / Python` is fine

Reply:
yeah